src/app/modules/main/chat_section/module.nim

```diff
@@ -576,7 +576,7 @@ proc updateChatLocked(self: Module, chatId: string) =
   let locked = self.controller.checkChatIsLocked(communityId, chatId)
   self.view.chatsModel().setItemLocked(chatId, locked)
 
-proc updateChatRequiresPermissions(self: Module, chatId: string) =
+proc updateChatRequiredPermissions(self: Module, chatId: string) =
```

Comment:
I think the previous spelling was ok. If meant that the "chat requires permisisons".
If we want to rename it, it could be `updatePermissionsRequiredOnChat`

Reply:
ah, I was reading it wrong, Previous spelling makes sense after second thought, but I like your proposal `updatePermissionsRequiredOnChat` as it clears the possible confusion 😄